src/libraries/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -149,22 +149,25 @@ public void SetAccessControl_FileStream_FileSecurity_InvalidArguments()
             Assert.Throws<ArgumentNullException>("fileStream", () => FileSystemAclExtensions.SetAccessControl((FileStream)null, fileSecurity: null));
         }
 
-        [Fact]
-        public void SetAccessControl_FileStream_FileSecurity_InvalidFileSecurityObject()
+        [Theory]
+        [InlineData(FileMode.Append)]
+        [InlineData(FileMode.Open)]
+        [InlineData(FileMode.OpenOrCreate)]
```

Comment:
This test asserts for ArgumentNullException when fileSecurity is null. Using different FileModes doesn't make the code go through another path, it throws the same exception in all cases. You should remove this InlineData or at least add it to a test where using different FileModes is not meaningless.